apps/heft/src/plugins/TypeScriptPlugin/internalTypings/TypeScriptInternals.ts

```diff
@@ -103,42 +94,6 @@ export interface IExtendedTypeScript {
     directoryExists: (path: string) => boolean
   ): string[];
 
-  /**
-   * https://github.com/microsoft/TypeScript/blob/5f597e69b2e3b48d788cb548df40bcb703c8adb1/src/compiler/emitter.ts#L261-L614
-   */
-  emitFiles(
-    resolver: IEmitResolver,
-    host: IEmitHost,
```

Comment:
This function was also the only use of types like IEmitHost, IEmitResolver, IEmitTransformers, etc, so can also be removed.